torch/_subclasses/fake_tensor.py

```diff
@@ -2319,7 +2319,6 @@ def _dispatch_impl(
         device_conversion_skip_const_prop = (
             func is torch.ops.aten._to_copy.default
             and isinstance(args[0], torch.Tensor)
-            and args[0].device.type == "meta"
```

Comment:
what's this change for?

Reply:
for 

cpu_tensor.to("cuda:0")

Reply:
I feel like we can either (1) make the check more sophisticated by checking if we have the device that we are calling to_copy on, or (2) delete this entire variable and use of it.

The variable is only being used to check if we can skip const propagation. We want to skip const propagate in the case where the the device we're calling to_copy on doesn't exist, because otherwise we'll get a device error. 

Or, we can just never const propagate it and always convert it to a fake tensor. I believe your change already effectively does this, and it seems like CI is ok, so maybe this is ok.

Reply:
We should only skip the const prop if the device we are converting to doesn't exist on this machine

Reply:
Actually, question: what's the test case that triggers the constant prop logic?

Reply:
cpu_tensor.to("cuda:0")